src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -87,30 +45,24 @@ class DataType(Enum):
     REDUCED = "Reduced"
 
 
+class ScanPosition(Enum):
+    BOTTOM = "Bottom"
+    TOP = "Top"
+    NONE = None
+
+
 @dataclass(frozen=True)
 class Block:
     block_type: str
     raw_lines: list[str]
 
     @staticmethod
     def create(reader: CsvReader) -> Block:
-        block_cls_by_type: dict[str, type[Block]] = {
-            "Group": GroupBlock,
-            "Note": NoteBlock,
-            "Plate": PlateBlock,
-        }
-
-        for key, cls in block_cls_by_type.items():
-            if reader.match(f"^{key}"):
-                return cls.create(reader)
-
-        error = f"Expected block '{reader.get()}' to start with one of {sorted(block_cls_by_type.keys())}."
-        raise AllotropeConversionError(error)
+        raise NotImplementedError
```

Comment:
Let's delete this method as well as `NoteBlock.create()`. It's no longer implemented by `PlateBlock` or any of its children, it's only called as `GroupBlock.create()`, and `NoteBlock.create()` isn't called.